packages/ember-routing/lib/location/history_location.js

```diff
@@ -101,7 +101,8 @@ export default EmberObject.extend({
     // remove baseURL and rootURL from start of path
     let url = path
       .replace(new RegExp(`^${baseURL}(?=/|$)`), '')
-      .replace(new RegExp(`^${rootURL}(?=/|$)`), '');
+      .replace(new RegExp(`^${rootURL}(?=/|$)`), '')
+      .replace(/\/\//g,'/'); // remove extra slashes
```

Comment:
lets just anchor at the end, we can always make it more permissive if we end up finding a need for it...